### DISCUSSION TITLE
Not working Google calendar on IOS devices

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [add2cal/add-to-calendar-button#423] [Q&A] [answered]
When I click to the Google Calendar It is opening the Google Calendar App but not setting the event for first time so I had to click twice and it is letting me to save the event. any solution?
@jekuer

**Comment by MuttakinHasib** [comment]
https://github.com/add2cal/add-to-calendar-button/assets/44552983/a48975f2-4800-4055-a07a-7285d1d05a00

**Comment by HasanCSE-IYI** [comment]
This is crucial! A user have to click on Google calendar twice to get there on iPhone. Should be fixed soon I hope.

**Comment by jekuer** [comment]
Unfortunately, as of now, I did not find any solution to mitigate this.
The problem is not this script, but the Google Calendar App, which is extremely (!) buggy under iOS (maybe due to Apple restrictions, maybe because Google does not put that much effort into it).
When the app is running in some background mode or is not open at all, any deep link will open the app and then get lost.
I tried opening the link 2 times, buts this has no effect either - maybe with a very big delay.

**Since this did not occur in the past, I would give it a try to wait for another update, but open for other ideas!**

**Comment by jekuer** [comment, marked as answer]
With latest version 2.4, there now is a workaround, telling the user about it. Not ideal, but best we can do atm until Google fixes it.

**Reply by guilherme-moretto-score** [reply]
Is it possible to remove the message displayed when Google Calendar is selected?

**Reply by jekuer** [reply]
@guilherme-moretto-score , did Google fix the issue?
I did not re-test it recently.

**Reply by guilherme-moretto-score** [reply]
@jekuer I guess they did, just tested on Iphone a few times and it went fine (I closed Google Calendar app before testing).

**Reply by jekuer** [reply]
Confirmed. They fixed it with the recent update. Will remove the workaround asap.

**Reply by jekuer** [reply]
Latest versions (>2.5) now no longer have the workaround included.
